go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/coreos/go-oidc/v3 v3.9.0
 	github.com/deckarep/golang-set/v2 v2.3.1
 	github.com/docker/go-connections v0.4.0
-	github.com/envoyproxy/go-control-plane v0.11.1
+	github.com/envoyproxy/go-control-plane v0.12.0
```

Comment:
Probably fine but notable. There’s a bunch of changes in https://github.com/envoyproxy/go-control-plane/releases/tag/v0.12.0, sadly they don’t seem to differentiate them now in the changelog, but nothing explicitly flagged as breaking.

Reply:
I reverted this in 2e06b92c43a11fa6419434a15fb4f519afb3cff4.  I'm not sure when this changed during the process.  Thank you for calling it out.